.github/workflows/ci-code-style.yml

```
@@ -9,10 +9,10 @@ on:
 jobs:
   unit_tests:
     name: Linter checks for KIWI python and Shell code
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

3.13 hasn't been released yet
```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13.0-rc.1"]
```

---

I fixed the 3.13 python version by using 3.13-dev